JapaneseFilter/sections/specific.txt

```diff
@@ -38,6 +38,7 @@ ai-myon.com,mikobito.com,chimolog.co,pokeca-new-card.com,useful-notes.com,otakuk
 admin.blog.fc2.com###ad_sidebody
 admin.blog.fc2.com##.column-left_ad
 koro-pokemon.com###sadr
+appbank.net##a[href="https://ja.appa.pe/"]
```

Comment:
Btw, it seems that the link of this image is stable.
We can remove it via `||appbank.net/wp-content/uploads/*/appape.png`.